ui/apps/platform/src/types/search.ts

```diff
@@ -17,3 +17,19 @@ export type SearchEntry = {
     value: string; // an option ends with a colon
     label: string; // an option ends with a colon
 };
+
+export type GlobalSearchOption = {
+    value: string;
+    label: string;
+    type?: string;
+};
+
+export type RestSortOption = {
+    field: string;
+    reversed: boolean;
+};
+
+export type GraphQLSortOption = {
+    id: string;
+    desc: boolean;
+};
```

Comment:
Since our UI is supposed to be the only consumer of our GraphQL API, we should submit a request to the API teams that they standardize on the REST style for this.